scripts/ci/proxy/nginx-proxy-tls-grpc.conf

```diff
@@ -10,7 +10,7 @@ server {
 
 server {
 	listen 8443 ssl http2;
-	server_name central-proxy.stackrox.local;
```

Comment:
Can you make this configurable that the parameter inserted from the outside is used as a server name here?
These scripts are very useful to quickly setup proxies for local testing.